docs/rancher/csi-driver.md

```diff
@@ -277,6 +277,121 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
 
 1. You can now create a PVC based on this new **StorageClass**, which utilizes the **Host StorageClass** to provision volumes on the bare-metal Harvester cluster.
 
+## RWX volumes support
+
+### Prerequisites
+
+:::note
+
+To use this feature, you need to ensure the host harvester cluster version is **v1.4.0** or later.
+
+:::
+
+Before creating RWX volumes on the guest cluster, you need to create a corresponding RWX StorageClass on the host cluster.
+
+1. Create a RWX StorageClass on the host
```

Comment:
```suggestion
- Harvester v1.4 or later is installed on the host cluster.

- You have created an RWX StorageClass on the host Harvester cluster.

  On the **Storage Class: Create** screen, click **Edit as YAML** and specify the following:
```

Reply:
The Prerequisites section usually consists of a plain list. I am replacing both the note and the procedure with an unordered list.